### ISSUE TITLE
single-letter navigation behaving improperly when "speech interrupt for typed characters" is disabled. h

### ISSUE THREAD

**Issue by mikolysz** [nvaccess/nvda#6677]
when "speech interrupt for typed characters" is disabled, navigating by single-letter navigation makes nvda work improperly. NVDA doesn't announce the item we're navigating to instantly when it's already speaking. Instead, it finishes what it has to say and announces the item. I find this behaviour unexpected and annoying. I'm not sure if it's a bug or a feature, if it's a feature, then a separate option to disable "speech interrupt for typed characters" but keep single-letter navigation behaving as normal could be added.

**Comment by leonardder** [comment]
IMO it is expected behaviour, but may be we should tie speech interrupt for typed characters to objects with _hasNavigableText set to True (thus, editable text cases)?

**Comment by feerrenrut** [comment]
At the moment I don't really understand the problem. @miki123211 Could you update this issue with an example of what NVDA speech you get with and without the "speech interrupt for typed characters" enabled. For each case give an example of the speech you expect to hear, and why this should be the case.

This will make it easier to understand, and hopefully this issue can be progressed further.

**Comment by Adriani90** [comment]
@feerrenrut my understanding is that the author wants to disable speech interrupt for typed characters only in edit fields while typing something, in focus mode or in text editors. But this should not apply to browse mode while using quick navigation keys.

**Comment by Adriani90** [comment]
We didn't hear any updates form the initial author upon request since may years. I am closing as abandoned. if you are still having this problem, please update the issue description as requested and we can reopen.
